### ISSUE TITLE
[INFO] Linux support is here!

### ISSUE THREAD

**Issue by GrowtopiaNoobs** [GrowtopiaNoobs/GrowtopiaServer#822]
I would like to announce that linux support is here! For now you can check on build commands in `appveyor.yml`, but I will add all informations in wiki soon. Also I would like to thank to @MarmoriPelaa for his work on it.

**Comment by kingawesomed2** [comment]
@GrowtopiaNoobs  i open private server with vps and my gt no work focusedsg said desired port port forwarder and no help can you help me?

**Comment by MarmoriPelaa** [comment]
Haha, though I am feeling a bit ironic. I'm having problems compiling bcrypt... ow-crypt.h or ow-crypt.cpp specifically. Different exception specifier in the `crypt(...)` function found in ow-crypt.h.. Have you found a fix for that yet?

**Comment by herothedev** [comment]
How then appveyor can build it, did u choose right linux distribution (ubuntu 18.04)

**Comment by MarmoriPelaa** [comment]
I have no idea, but I just added a throw() in the crypt(...) function and it now works fine.

Also, I'm now struggling with the clothing stuff. For that I'd try to replace the line endings with Unix format using Notepad++

**Comment by herothedev** [comment]
Okay. So can't wear clothes still alive?

**Comment by GrowtopiaNoobs** [comment]
@MarmoriPelaa It builds on Ubuntu 18 fine (I builed it locally too), this error happened on Ubuntu 16. If you have found what changes are needed to make it work on Ubuntu 16, then please open new PR.

If the issue with clothes is caused by `getline`, then I think the best fix is to use some some `getline` alternative which support all line endings.

**Comment by herothedev** [comment]
@GrowtopiaNoobs @MarmoriPelaa I can wear clothes and its works on ubuntu 18.04 vps.

**Comment by sanderDijkxhoorn** [comment]
Doesn’t work for me

**Comment by WoofHax** [comment]
Centos 8 Linux OS is working too with wear item clothes 👌

**Comment by atomiclucky** [comment]
![image](https://user-images.githubusercontent.com/49339987/76153738-dfabc480-60d8-11ea-98ae-c987fa3ed2f4.png)
 ubuntu 16.04
kernel 2.6.32-042stab123.9
doesn't work

**Comment by MoanHawks** [comment]
cd enet2 
cp libenet.a ../libenet.a
cd ..

Then try build one more time @atomiclucky

**Comment by atomiclucky** [comment]
> 
> 
> cd enet2
> cp libenet.a ../libenet.a
> cd ..
> 
> Then try build one more time @atomiclucky

doesn't solve the problem. I guess I need to compile it on ubuntu 18.04

**Comment by MoanHawks** [comment]
Do you install cmake?

**Comment by atomiclucky** [comment]
yes I did

**Comment by atomiclucky** [comment]
I will install centos 7 ok dude?

**Comment by MoanHawks** [comment]
If you using centos 

Don't use sudo install build-essential to install devs tools 

Try use:
sudo yum groupinstall 'Development Tools'

**Comment by atomiclucky** [comment]
yeah I know that centos has the "yum" package manager, not the debian one

**Comment by MoanHawks** [comment]
k

**Comment by atomiclucky** [comment]
damn even on centos 
![image](https://user-images.githubusercontent.com/49339987/76166766-b6874480-6169-11ea-81ac-32a1a2c9a260.png)

**Comment by MoanHawks** [comment]
Did you install cMake????

**Comment by atomiclucky** [comment]
YES I DID 
![image](https://user-images.githubusercontent.com/49339987/76166838-5775ff80-616a-11ea-86e9-34c47351f51d.png)

**Comment by MoanHawks** [comment]
Can you do gcc --verison?  Or gcc -v

**Comment by atomiclucky** [comment]
gcc version 4.8.5 20150623 (Red Hat 4.8.5-39) (GCC)

**Comment by MoanHawks** [comment]
Not sure but my building system is clang v 9.0.0 I think you gcc is old

**Comment by atomiclucky** [comment]
ok my gcc is now 8.3.1

**Comment by MoanHawks** [comment]
Not sure bcz I am not using Ubuntu
